docs/api.md

```diff
@@ -95,7 +100,9 @@ The follow error responses include additional parameters:
 
 * errno 111:  a `serverTime` parameter giving the current server time in seconds.
 * errno 114:  a `retryAfter` parameter indicating how long the client should wait before re-trying.
-* errno 120:  a `email` parameter indicating the case used to create the account
+* errno 120:  an `email` parameter indicating the case used to create the account.
+* errno 130:  a `region` parameter indicating the region code that was deemed invalid.
+* errno 132:  `reason` and `reasonCode` parameters indicating the reason that the message was rejected.
```

Comment:
It might be useful either here or below to link to nexmo's docs for `reason` and `reasonCode`.